src/v2/Apps/FairOrginizer/fairOrganizerRoutes.tsx

```diff
@@ -29,9 +31,22 @@ export const fairOrganizerRoutes: AppRouteConfig[] = [
     prepare: () => {
       FairOrganizerApp.preload()
     },
+    render: ({ Component, props }) => {
+      if (Component && props) {
```

Comment:
**Question:** is there a case when we wouldn't have component or props, so this would fail? If so, what happens?

Reply:
@mdole those data just come from the server side and `render` method may be executed without `props` for the first time. And in this case nothing is rendered 🤷‍♀️